### ISSUE TITLE
Improve docs for subtensor

### ISSUE THREAD

**Issue by steffencruz** [opentensor/bittensor-subnet-template#40]
Lots of confusion about how to run subtensor locally & also the meaning and significance of various chains.

@Eugene-hu @rajkaramchedu let's plan for improving this

**Comment by rajkaramchedu** [comment]
Yes. I have an open issue in the docs repo: https://github.com/opentensor/new-docs/issues/47
